.github/workflows/build_and_test.yml

```diff
@@ -110,6 +119,11 @@ jobs:
           script_args+=(--sccache-credentials $(basename ${GOOGLE_GHA_CREDS_PATH}))
         fi
 
+        if [ ${{ inputs.RUNS_ON }} == 'self-hosted' ]; then
+          RUNNER_CERTIFICATES_DIR=/etc/pki/ca-trust/source/anchors/
+          mkdir -p ${GITHUB_WORKSPACE}/certificates
+          cp ${RUNNER_CERTIFICATES_DIR}/*.crt* ${GITHUB_WORKSPACE}/certificates
+        fi
```

Comment:
Could we use the `DOCKER_RUN_ARGS` input to add `-v /etc/pki/ca-trust/source/anchors/.../bundle.crt:/etc/pki/ca-trust/.../llnl.bundle.crt:ro`.

That way the certificate would be at the proper location in read-only, with no copy required from within the container.

Then inside of the docker container simply update the certificates (we can do it whatever the case : it's fast and could do not harm?). Or if we really want it, we can add an option.

Reply:
We certainly can...and probably should. Would you mind if we did this in a follow up PR. We are getting another workstation to run integratedTests on the CPU, and we will require a generic treatment at that time. I would rather just get this merged as is, and in separate PR's:

1.  clean up and generalize for the new workstation,
2. fix the failing unit tests on GPU, and
3. get integrated tests diffs worked out.

we can do all of these steps independent of each other IMO.

Reply:
OK, I'm good with it. We need to be sure we actually have time to revisit this.